traffic_ops/traffic_ops_golang/traffic_ops_golang.go

```diff
@@ -61,11 +63,31 @@ func main() {
 	}
 
 	if err := log.InitCfg(cfg); err != nil {
-		fmt.Println("Error initializing loggers: %v", err)
+		fmt.Printf("Error initializing loggers: %v\n", err)
 		return
 	}
 
-	log.Infof("Using Config: %+v\n", cfg)
+	log.Infof(`Using Config values:
+		Port:                %s
+		Db Server:           %s
+		Db User:             %s
+		Db Name:             %s
+		Db Ssl:              %t
+		Max Db Connections:  %d
+		TO URL:              %s
+		Insecure:            %t
+		Cert Path:           %s
+		Key Path:            %s
+		Proxy Timeout:       %d
+		Read Timeout:        %d
+		Read Header Timeout: %d
+		Write Timeout:       %d
+		Idle Timeout:        %d
+		Error Log:           %s
+		Warn Log:            %s
+		Info Log:            %s
+		Debug Log:           %s
+		Event Log:           %s`, cfg.HTTPPort, cfg.DBServer, cfg.DBUser, cfg.DBDB, cfg.DBSSL, cfg.MaxDBConnections, cfg.TOURLStr, cfg.Insecure, cfg.CertPath, cfg.KeyPath, cfg.ProxyTimeout, cfg.ReadTimeout, cfg.ReadHeaderTimeout, cfg.WriteTimeout, cfg.IdleTimeout, cfg.LogLocationError, cfg.LogLocationWarning, cfg.LogLocationInfo, cfg.LogLocationDebug, cfg.LogLocationEvent)
```

Comment:
What about using a nested struct instead, here? It's a lot less code, and people don't have to remember to add things here too.

https://play.golang.org/p/iO2T96zIQI